front/src/app/components/common/order-filter-panel/index.less

```diff
@@ -0,0 +1,86 @@
+@import 'guide.less';
+
+@orderFilterPanel-columnWidth: minmax(140px, 200px);
+@orderFilterPanel-columnGap: 10px;
+
+// Mixins
+
+.order-filter-panel__footer {
+    margin: 50px 0 0 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+// Component
+
+.order-filter-panel {
+    // grid
+    display: grid;
+    grid-template-columns: repeat(2, @orderFilterPanel-columnWidth);
+    grid-column-gap: @orderFilterPanel-columnGap;
+    height: 100%;
+
+    // font
+    font-family: @font-family;
+    font-size: 15px;
+    font-weight: 400;
+    color: @color-font-dark;
+
+    &__column1 {
```

Comment:
it is not BEM

colum1 is not element. it looks like col1/xs-col2/etc. from bootstrap.

do not fix it now